tutor/templates/apps/openedx/settings/lms/development.py

```diff
@@ -24,7 +24,11 @@
 # CMS authentication
 IDA_LOGOUT_URI_LIST.append("http://{{ CMS_HOST }}:8001/logout/")
 
-FEATURES['ENABLE_COURSEWARE_MICROFRONTEND'] = False
+FEATURES["ENABLE_COURSEWARE_MICROFRONTEND"] = False
+
+# Disable enterprise integration
+FEATURES["ENABLE_ENTERPRISE_INTEGRATION"] = False
```

Comment:
I [ran into this issue](https://github.com/openedx/frontend-app-authn/issues/692) myself, but later could not reproduce it: I assumed it was because of the CSRF issue we fixed.

Now I realize it wasn't: I was just testing via dev, and later retested via local.  Made me think it was gone.  Nice catch.